frontend-v2/src/features/data/Data.tsx

```diff
@@ -52,6 +54,18 @@ const Data:FC = () => {
     };
   }
 
+  function downloadCSV() {
+    const csv = generateCSV(dataset, groups, subjectBiomarkers, units);
+    console.log(csv)
+    const blob = new Blob(['\ufeff', csv], { type: 'text/csv' });
```

Comment:
'\ufeff` here is the [Unicode BOM](https://en.wikipedia.org/wiki/Byte_order_mark). Without it, unicode characters get mangled during the download.